src/lib/components/ui/Modal/README.mdx

```diff
@@ -283,7 +283,7 @@ component.
               }]}
               autoFocus={false}
               closeHandler={() => setModalOpen(false)}
-              scrollMode={modalScrollMode}
+              scrollView={modalScrollMode === 'body' ? undefined : null}
```

Comment:
I just don't get it. According to `propTypes`, this can only be a `node` (or `undefined`). What I see here is `undefined` (OK, omit the attribute and use the default value) and `null` (…?). Can somebody please explain to me what's going on here? 👀

Reply:
For the sake of clarity, I'd rename the effect in this example to `modalScrollView` and make it boolean (also see my confusion below). The previously intended extensibility of `modalScrollMode` is no longer valid with the API change here in this PR.

Reply:
> this can only be a node (or undefined)

The `ScrollView` prop is not required. It can therefore be `node`, `null` or `undefined`. They mean the following things:
* `node` use the specified instance of `ScrollView`
* `undefined` do not use any `ScrollView`, scroll the entire modal including header and footer
* `null` use the instance of `ScrollView` defined in `defaultProps`

Reply:
> I'd rename the effect in this example to modalScrollView

done